src/typescript/frontend/src/components/pages/metadata/MetadataPage.tsx

```diff
@@ -57,42 +72,69 @@ const MetadataPage = () => {
-            fields.delete(key)
+            fields.delete(key);
             setFields(new Map(fields));
           }}
           buttonText="-"
-        >
-        </FormEntry>
+        ></FormEntry>
       ))}
       <FormEntry
         name="Add new field"
         value={addFieldName}
         placeholder="New field name"
         onChange={(e) => {
-          setAddFieldName(e.target.value)
+          setAddFieldName(e.target.value);
         }}
         onClick={() => {
           setFields(new Map(fields.set({ name: addFieldName, placeholder: addFieldName }, "")));
           setAddFieldName("");
         }}
         buttonText="+"
-      >
-      </FormEntry>
+      ></FormEntry>
       <div>{/* For spacing. */}</div>
       <div className="mx-auto w-[300px] grid grid-cols-2 grid-rows-2 gap-[10px]">
-        <button tabIndex={3} className={`${BUTTON_CLASSNAME}`} onClick={() => {
-          const data = {
-            marketAddress,
-            fields: Array.from(fields.entries()),
-          };
-          navigator.clipboard.writeText(JSON.stringify(data));
-        }}>Copy</button>
-        <button tabIndex={3} className={`${BUTTON_CLASSNAME}`} onClick={() => {
-          navigator.clipboard.readText().then(r => { const data = JSON.parse(r); setFields(new Map(data.fields)); setMarketAddress(data.marketAddress); }).catch(console.warn);
-        }}>Paste</button>
-        <button tabIndex={3} className={`${BUTTON_CLASSNAME} ${account === null ? "!border-dark-gray" : ""} col-span-full`} onClick={() => {
-          navigator.clipboard.readText().then(r => setFields(new Map(JSON.parse(r)))).catch(console.warn);
-        }} disabled={account === null}>Submit</button>
+        <button
+          tabIndex={3}
+          className={`${BUTTON_CLASSNAME}`}
+          onClick={() => {
+            const data = {
+              marketAddress,
+              fields: Array.from(fields.entries()),
+            };
+            navigator.clipboard.writeText(JSON.stringify(data));
+          }}
+        >
+          Copy
+        </button>
+        <button
+          tabIndex={3}
+          className={`${BUTTON_CLASSNAME}`}
+          onClick={() => {
+            navigator.clipboard
+              .readText()
+              .then((r) => {
+                const data = JSON.parse(r);
+                setFields(new Map(data.fields));
+                setMarketAddress(data.marketAddress);
+              })
+              .catch(console.warn);
+          }}
+        >
+          Paste
+        </button>
+        <button
+          tabIndex={3}
+          className={`${BUTTON_CLASSNAME} ${account === null ? "!border-dark-gray" : ""} col-span-full`}
+          onClick={() => {
+            navigator.clipboard
+              .readText()
+              .then((r) => setFields(new Map(JSON.parse(r))))
+              .catch(console.warn);
+          }}
+          disabled={account === null}
+        >
+          Submit
+        </button>
```

Comment:
@CRBl69 I didn't actually change anything in this file, it's entirely formatting changes from `pnpm run format`